### ISSUE TITLE
A11y: Focus lost after closing the quickview and offcanvas cart

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#7441]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.10.2  | *Affected area / extension:* Extension:CMS-Professional 

 ----

*_Actual behaviour:_*

If you close the quickview you have to navigate through the whole page again.

*_Expected behaviour:_*

After closing the quickview the focus should be resumed at the product box.
Edit: The focus should also be resumed after closing the offcanvas cart when adding the product to the cart via quickview.

*_How to reproduce:_*

1. Enable Quickview in your Listings
2. Open and close quickview via keyboard

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CMS-502**



[created from CMS-502, comment 544401]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from CMS-502, comment 544478]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from CMS-502, comment 544483]

**Comment by marcelbrode** [comment]
Fixed by:
- https://github.com/shopware/SwagCmsExtensions/pull/37
- https://github.com/shopware/SwagCmsExtensions/pull/38
